include/z64.h

```diff
@@ -777,16 +777,19 @@ typedef struct {
     /* 0x08 */ DmaRequest  dmaRequest;
     /* 0x28 */ OSMesgQueue loadQueue;
     /* 0x40 */ OSMesg   loadMsg;
-} ObjectStatus; // size = 0x44
+} ObjectLoadEntry; // size = 0x44
+
+// 1000 kilobytes
+#define OBJECT_SPACE_SIZE_BASE (1000 * 1024)
```

Comment:
I suggest having this define in `z_scene.c`, maybe next to the function where it's used, since it's not intended for use externally I'd say